app/routes/resources+/note-editor.tsx

```diff
@@ -14,16 +14,15 @@ import { redirectWithToast } from '~/utils/flash-session.server.ts'
 
 export const NoteEditorSchema = z.object({
 	id: z.string().optional(),
-	title: z.string().min(1),
-	content: z.string().min(1),
+	title: z.string(),
+	content: z.string(),
```

Comment:
I seem to remember talking about this, but I forgot. Does this mean that users will be able to create notes with no title or content? Or is `min(1)` assumed? If it's assumed, then I would need to do `z.string().optional()` if I wanted to allow an empty string? I think that makes sense in this context, but I just wanted to make sure my understanding is correct.

Reply:
It's adhere to how zod works now which means all fields are required by default. 

>  I would need to do z.string().optional() if I wanted to allow an empty string
Exactly